npc/json.go

```diff
@@ -4,7 +4,56 @@ import (
 	"encoding/json"
 )
 
+// Return JSON suitable for logging an API object.
 func js(v interface{}) string {
+	// Get the raw JSON
 	a, _ := json.Marshal(v)
+	// Convert this back into a tree of key-value maps
+	var m map[string]interface{}
+	if err := json.Unmarshal(a, &m); err != nil {
```

Comment:
Is it possible to avoid `json.Marshal` and `json.Unmarshal`, and just directly cast `v`?

Reply:
No; `v` is expected to be a strongly-typed Go object, like a `Pod`